src/components/App.js

```diff
@@ -9,6 +9,8 @@ const { getSources, getSelectedSource } = require("../selectors");
 const { KeyShortcuts } = require("devtools-sham-modules");
 const shortcuts = new KeyShortcuts({ window });
 
+const cmd = `${cmdString()}+P`;
```

Comment:
This value is per-OS so it won't change during runtime and really should be a const instead of evaluated in the render function.